fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -100,4 +106,27 @@ public void testGetTimeMapResponse() throws IOException {
             assertArrayEquals(expectedLinks, bodyLinks);
         }
     }
+
+    @Test
+    public void testGetTimeMapRDFSubject() throws IOException {
+        final String id = getRandomUniqueId();
+        final String subjectURI = serverAddress + id;
+
+        final HttpPut createMethod = putObjMethod(id);
+
+        createMethod.addHeader(CONTENT_TYPE, "text/n3");
+        createMethod.addHeader(LINK, VERSIONED_RESOURCE_LINK_HEADER);
+        createMethod.setEntity(new StringEntity("<" + subjectURI + "> <info:test#label> \"foo\""));
+
+        try (final CloseableHttpResponse response = execute(createMethod)) {
+            assertEquals("Didn't get a CREATED response!", CREATED.getStatusCode(), getStatus(response));
+        }
+        final HttpGet httpGet = getObjMethod(id + "/fcr:versions");
```

Comment:
May as well use the constant: `FedoraType.FCR_VERSIONS`.

Reply:
Ooops, good catch